### PR TITLE
Check if state transitions exist at test time

### PR DESCRIPTION
**Pull request by Porges**
Don't ignore missing state transitions in `StatefulOrm` static constructor test, instead error out. 

`Scaleset` was missing the `Resize` transition.

### CODE REVIEW

File: src/ApiService/ApiService/onefuzzlib/orm/Orm.cs [hunk 1]
```diff
@@ -178,9 +180,12 @@ public class StatefulOrm<T, TState, TSelf> : Orm<T>, IStatefulOrm<T, TState> whe
                     continue;
                 }
 
-                throw new Exception($"State transition method '{state}' in '{thisType?.Name}' does not have the correct signature. Expected '{delegateSignature}'  actual '{methodInfo}' ");
+                throw new InvalidOperationException($"State transition method '{state}' in '{thisType.Name}' does not have the correct signature. Expected '{delegateSignature}'  actual '{methodInfo}' ");
             };
 
+            if (missing.Any()) {
```
Comment (chkeita) [inline src/ApiService/ApiService/onefuzzlib/orm/Orm.cs:186]: Not all state have a corresponding transition method.
ex: JobState.Enabled and JobState.Stopped do not have a transition method in jobs.py

Comment (Porges) [reply src/ApiService/ApiService/onefuzzlib/orm/Orm.cs:186]: > Not all state have a corresponding transition method. ex: JobState.Enabled and JobState.Stopped do not have a transition method in jobs.py

I think that it's probably a good idea that we explicitly have these as no-op transitions, so we can check them all?

Comment (chkeita) [reply src/ApiService/ApiService/onefuzzlib/orm/Orm.cs:186]: sounds good

**Review by tevoinea — APPROVED** [review]

File: src/ApiService/ApiService/onefuzzlib/NodeMessageOperations.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+//# being in onefuzztypes
+public record NodeMessage(
+    [PartitionKey] Guid MachineId,
+    [RowKey] string MessageId,
+    NodeCommand Message
+) : EntityBase {
+    public NodeMessage(Guid machineId, NodeCommand message) : this(machineId, NewSortedKey, message) { }
+};
+
+public interface INodeMessageOperations : IOrm<NodeMessage> {
```
Comment (Porges) [inline src/ApiService/ApiService/onefuzzlib/NodeMessageOperations.cs:16]: Moved this type and NodeTasksOperations into their own files.

**Review by stishkin — APPROVED** [review]

**Pull request merged**